awx/ui_next/src/components/LaunchPrompt/useSteps.js

```diff
@@ -13,14 +13,20 @@ export default function useSteps(config, resource, i18n) {
     useOtherPromptsStep(config, resource, visited, i18n),
     useSurveyStep(config, resource, visited, i18n),
   ];
+  const formErrors = steps
+    .filter(s => s.formError != null)
+    .filter(s => s !== undefined)
+    .filter(s => Object.keys(s.formError).length > 0);
+
+  const formErrorsContent = [];
```

Comment:
you don't need to separate these... just tack on a map to your filter statement.  I believe you could simplify to:

```
  const formErrorsContent = steps
    .filter(s => s?.formError && Object.keys(s.formError).length > 0) // the first part is equivalent to s !== undefined && s.formError != null
    .map(({formError}) => formError)
```